examples/kind/main.tf

```diff
@@ -127,7 +126,7 @@ module "minio" {
   cluster_issuer   = local.cluster_issuer
   argocd_namespace = module.argocd_bootstrap.argocd_namespace
 
-  enable_service_monitor = false
+  enable_service_monitor = local.enable_service_monitor
```

Comment:
We should add it to `locals.tf`

Reply:
I forgot that when integrating this change from my personal example. Good catch ;)